11.03-Cotributed-Example

shijiale0609 opened the pull request:
11.03 talk about Toy Example of Kennedy-O'Hagan Model in the book. The code is mainly from the supplementary materials with some modifications and corrections.

shijiale0609 committed:
11.03-Cotributed-Example

Toy Example of Kennedy-O'Hagan Model in the book

adowling2 committed:
Fixed section headers.

adowling2 committed:
Verified notebook runs on local computer.

adowling2 commented:
@shijiale0609 Thank you for this contribution. I made a few edits to the style.

In its current form, the notebook is difficult to follow. Please add:
- Meaningful subsection titles to streamline the presentation. What are the key steps in the analysis? You can make these subsection titles.
- A little bit of text between sections of code.
- Comments in the code blocks.

A vast majority of the other notebooks presented this semester use all three of the above-mentioned techniques to improve clarity. Please see the following for examples:
* https://ndcbe.github.io/cbe67701-uncertainty-quantification/10.01-Contributed-Example.html
* https://ndcbe.github.io/cbe67701-uncertainty-quantification/09.01-Contributed-Example.html
* https://ndcbe.github.io/cbe67701-uncertainty-quantification/04.01-Local-sensitivity-analysis.html

To make changes, you should first download the updated version of the notebook, available here:

https://github.com/ndcbe/cbe67701-uncertainty-quantification/blob/chap11-Jiale/notebooks/11.03-Contributed-Example.ipynb

After making the changes, you can then reupload to GitHub. Use the same branch `chap11-Jiale`. You do not need to open a new pull request.

shijiale0609 committed:
improve clarity

1. add section & subsection
2. add text between codes
3. add comments in the code blocks

shijiale0609 committed:
improve clarity-2

modify the subsection

shijiale0609 committed:
update the subsection

1. update the subsections for the "plot surface"

adowling2 committed:
Added legends.
